helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java

```diff
@@ -93,10 +93,10 @@ public class MaintenanceManagementService {
   private final HelixDataAccessorWrapper _dataAccessor;
   private final Set<String> _nonBlockingHealthChecks;
   private final Set<StoppableCheck.Category> _skipHealthCheckCategories;
-  // Set the default value of _stoppableHealthCheckList to be the list of all stoppable checks to
+  // Set the default value of _skipStoppableHealthCheckList to be an empty list to
   // maintain the backward compatibility with users who don't use MaintenanceManagementServiceBuilder
   // to create the MaintenanceManagementService object.
-  private List<HealthCheck> _stoppableHealthCheckList = HealthCheck.STOPPABLE_CHECK_LIST;
+  private List<HealthCheck> _skipStoppableHealthCheckList = Collections.emptyList();
```

Comment:
This is for users that didn't use `MaintainenceServiceBuilder` to `MaintainenceService`. Since I didn't modify the existing constructors, they can't set `_skipStoppableHealthCheckList ` in their code, which means `_skipStoppableHealthCheckList` has have a default value.